spec/reservation.spec.js

```diff
@@ -10,7 +10,9 @@ describe('Reservations Testing Server:', () => {
           expect(res.status).to.equal(200);
           expect(res.data).to.be.an('object');
           expect(res.data).to.have.property('available')
-          expect(res.data.available).to.be.exist;
+          expect(res.data.available).to.exist;
+          expect(res.data).to.have.property('reservationId');
+          expect(res.data.reservationId).to.exist;
```

Comment:
Why are you checking for the reservationId twice?

Reply:
Oops that is suppose to check that its a string